### DISCUSSION TITLE
Animation graph pauses when character is off camera

### DISCUSSION THREAD

**Discussion by sommerlost** [Sixze/ALS-Refactored#251] [Q&A]
I've been working on a cinematic camera system for multiplayer seating and realized that whenever the character is completely off camera for about 1 second the animation graph stops running (as active) until the character is back in view of a camera.

I'm curious if there is something in the ALS code causing characters to pause when off screen for performance. Any insights would be greatly appreciated. Thanks!

**Comment by ameaninglessname** [comment]
It should be the "tick option" property on mesh component
